src/main/groovy/io/openliberty/tools/gradle/tasks/DevTask.groovy

```diff
@@ -250,6 +251,16 @@ class DevTask extends AbstractServerTask {
         this.keepTempDockerfile = keepTempDockerfile;
     }
 
+    @Optional
+    @Input
+    Boolean generateFeatures;
+
+    // Need to use a string value to allow someone to specify --generateFeatures=false
```

Comment:
Is the `generateFeatures` Boolean defaulting to `true` like Maven? Cannot tell from this code.

Reply:
Yes, see https://github.com/OpenLiberty/ci.gradle/blob/generateFeatures/src/main/groovy/io/openliberty/tools/gradle/tasks/DevTask.groovy#L83 and https://github.com/OpenLiberty/ci.gradle/blob/generateFeatures/src/main/groovy/io/openliberty/tools/gradle/tasks/DevTask.groovy#L952-L954

Added a comment to clarify this in https://github.com/OpenLiberty/ci.gradle/pull/676